itunespodcast/build.gradle

```diff
@@ -9,6 +9,7 @@ dependencies {
     testAnnotationProcessor mn.micronaut.inject.java
 
     testImplementation(libs.groovy.xml)
+    testImplementation(mn.micronaut.jackson.databind)
```

Comment:
can we use serde instead?

Reply:
Done

Reply:
should this be ` testImplementation(mn.micronaut.serde.jackson)`?